torchft/_torchft.pyi

```diff
@@ -59,3 +60,42 @@ class LighthouseServer:
     ) -> None: ...
     def address(self) -> str: ...
     def shutdown(self) -> None: ...
+
+@dataclass
+class PyQuorumMember:
```

Comment:
Can we avoid the Py* prefixes for these? If you're using them in Python you already know they're Python

Reply:
This is to avoid duplicate naming with the one from protobuffer. Let me try to see if it works or not.